package.json

```diff
@@ -64,7 +64,7 @@
     "node": ">=6"
   },
   "devDependencies": {
-    "@types/node": "^8.5.8",
+    "@types/node": "^9.4.6",
```

Comment:
Should we update also this?

Reply:
it's a devDependency, I don't see an issue.

Reply:
Tests didn't fail on my end.